lucid/launcher.py

```diff
@@ -215,7 +224,8 @@ def launch(beamline, *, toolbar=None, row_group_key="location_group",
     cbs = [grid.add_from_dict]
     loader = HappiLoader(beamline=beamline,
```

Comment:
A flag to a loader saying "don't do anything" seems a bit of an anti-pattern (a separate `NoOpLoader` might have been cleaner?)

Reply:
Good point 👍